vsphere/tests/test_event.py

```diff
@@ -501,3 +511,21 @@ def test_include_events_empty_event_resource_filters(aggregator, realtime_instan
     dd_run_check(check)
 
     assert len(aggregator.events) == 0
+
```

Comment:
Can we have a test for if the customer enables vsan but disables event collection? And if a customer has event filters in place and also enables vsan?

Reply:
The expected behavior should be that disabling event collection disables vsan and vsphere events. However, vsphere event filters are independent of vsan events (so as long as event collection is enabled and vsan is enabled, all of the hardcoded vsan event types will be reported).